Signal/src/OWSMessagesBubbleImageFactory.swift

```diff
@@ -5,6 +5,7 @@
 import Foundation
 import JSQMessagesViewController
 import SignalServiceKit
+import SignalMessaging
```

Comment:
In the course of doing this work, I moved `OWSSwiftUtils` to `SignalMessaging` so that we can use `owsFail()`, etc. in `SignalMessaging` and the SAE.  Unfortunately this means we need to explicitly import `SignalMessaging` in most of our .swift files.